### PR TITLE
Master training user assigned districts

### PR DESCRIPTION
**Pull request by sainivikas**
New field added on trainingUser model and changed get_user_mediators for send data to training app based district of trainer.

### CODE REVIEW

File: training/models.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from django.contrib.auth.models import User
 
 from django.db import models
-from geographies.models import State
+from geographies.models import State,District
```
Comment (goeltanmay) [inline training/models.py]: add space between , and District

File: training/models.py [hunk 2]
```diff
@@ -11,52 +11,55 @@ class TrainingUser(models.Model):
     id = models.AutoField(primary_key=True)
     user = models.OneToOneField(User, related_name="training_user")
     states = models.ManyToManyField(State)
-
```
Comment (goeltanmay) [inline training/models.py]: add blank line before __unicode__

File: training/models.py (hunk 1 above)
Comment (sainivikas) [inline training/models.py]: space added.

File: training/models.py (hunk 2 above)
Comment (sainivikas) [inline training/models.py]: new line added

**Pull request merged**